### ISSUE TITLE
Device properties

### ISSUE THREAD

**Issue by NubeDev** [NubeIO/driver-bacnet#131]
Device properties showing errors

https://sourceforge.net/projects/yetanotherbacnetexplorer/

Using YABE
![IMG_20241014_113102](https://github.com/user-attachments/assets/4e4971cd-4b43-4ae0-8da1-2850e564c07e)

And Niagara
![image](https://github.com/user-attachments/assets/77318f4b-2d21-4c14-b354-e94b11ba0d94)

**Comment by NubeDev** [comment]
the issue maybe related to as we dont seem to support

![image](https://github.com/user-attachments/assets/27d06835-51aa-4814-ab98-535d8d0f7a26)

for example my deviceID is `15210`
```
./bacrp 15210 8 15210 8
```

**Comment by shomaglasang** [comment]
Mate @NubeDev , the property PROP_ALL (8) is not supported yet.
Do you know the exact type/attribute of this property? Is this like an object list PROP_OBJECT_LIST?

**Comment by NubeDev** [comment]
Like i said you need to support like in `bacnet-stack/bin/bacserv` 

```
user@user:~/code/bacnet/bacnet-stack/bin$ ./bacserv 1234
BACnet Server Demo
BACnet Stack Version 1.3.8
BACnet Device ID: 1234
Max APDU: 1476

```

**Comment by shomaglasang** [comment]
@NubeDev The output above is a plain string (with new lines). I will implement it with the type "string". I tried to search the web if someone has implemented it but can't find any.

**Comment by shomaglasang** [comment]
mate @NubeDev , closing this issue.
Fix is included in 1.0.2-rc.4
https://github.com/NubeIO/driver-bacnet/releases/tag/v1.0.2-rc.4
